pages/pridejse.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Přidej se
+permalink: /pridejse
+---
+
+# Přidej se k nám!
+
+Studuješ na Fakultě informačních technologií a chceš poznat aktivní FIŤáky nebo jenom najít odreagování od Progtestu? 
+
+Tak jsi tady správně!
+
+Jsme klub FIT++, který působí na Fakultě informačních technologií ČVUT. Pořádáme akce jako [Hello FIT](https://klubfitpp.cz/hellofit), FIT párty, deskovky na fakultě, FITripy, LAN párty a spoustu další edukačních nebo zábavných eventů, na kterých se můžeš nejenom dozvědět knowlage nad rámec předmětů ve škole, ale hlavně poznat spoustu aktivních lidí z fakulty.
+
+**Proč bych se měl\* přidat?**
```

Comment:
Teď to asi není důležité, ale kapitoly bych pak zviditelnil přes markdown nadpisy (úroveň 2 nebo 3, idk.. cokoliv), z toho screenu cos posílal mi stránka přišla hodně... plochá? hodně textu, málo výrazného.. čehokoliv.